src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/appointments/AppointmentsResource.kt

```diff
@@ -49,7 +49,7 @@ class AppointmentsResource(private val appointmentService: AppointmentService) {
         responseCode = "201",
         description = "Appointment information with created id",
         content = [
-          Content(mediaType = "application/json", schema = Schema(implementation = CreateAppointmentRequest::class)),
+          Content(mediaType = "application/json", schema = Schema(implementation = CreateAppointmentResponse::class)),
```

Comment:
some swagger typos

Reply:
odd one, I've found that sometimes you don't need to specify the response and it works it out automatically, but don't really understand when and why that is